### ISSUE TITLE
If don't call provide in decl process wait for infinity

### ISSUE THREAD

**Issue by birhoff** [ymaps/modules#53]
Main problem is that you don't know that. And all what you can do is check all your code. Maybe you can add some stats method to call when something go wrong.

**Comment by dfilatov** [comment]
`modules.getStat()` can help you detect what module causes a problem.

**Comment by birhoff** [comment]
Yeah, this is what i wont, maybe add FAQ section where describe this method?

**Comment by zxqfox** [comment]
`won't` or `want`?

**Comment by birhoff** [comment]
of course want=) sorry for misspell.
